<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformComponentView</class>
 <widget class="QWidget" name="TransformComponentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TransformComponentView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="rotX">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="rotY">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="posY">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="scaleX">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Rotation:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="scaleY">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Scale:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="posX">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="posZ">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="rotZ">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="scaleZ">
     <property name="minimum">
      <double>-999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>posX</tabstop>
  <tabstop>posY</tabstop>
  <tabstop>posZ</tabstop>
  <tabstop>rotX</tabstop>
  <tabstop>rotY</tabstop>
  <tabstop>rotZ</tabstop>
  <tabstop>scaleX</tabstop>
  <tabstop>scaleY</tabstop>
  <tabstop>scaleZ</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
